### ISSUE TITLE
Subfile and main file can't have same file name

### ISSUE THREAD

**Issue by June-6th** [gsalzer/subfiles#4]
```
project
├── main.tex
└── subproject
    └── main.tex
```

project/main.tex:

```tex
\documentclass{article}
\usepackage{subfiles}
\begin{document}
\subfile{subproject/main.tex}
\end{document}
```

project/subproject/main.tex:

```tex
\documentclass[../main.tex]{subfiles}
\begin{document}
this is subproject/main.tex.
\end{document}
```

Can't compiled, but if can be compiled after changing `subproject/main.tex`'s file name.

**Comment by gsalzer** [comment]
Thanks for reporting the bug.

**Comment by gsalzer** [comment]
@June-6th 
The issue is addressed in the new version 2.0 of the package, available from github and, shortly, from CTAN.
Please let me know if you encounter further problems; in this case, open a new issue with sample code.
Thanks!
